src/System.Device.Gpio/System/Device/Gpio/GpioPin.cs

```diff
@@ -17,6 +17,14 @@ public class Gpio​Pin
             _pinNumber = pinNumber;
         }
 
+        /// <summary>
+        /// Only for compatibility reason and heritage. Do not use in normal usage.
+        /// </summary>
+        protected GpioPin()
```

Comment:
If we have this as protected, shouldn't the other ctor also be protected? Apparently subclassing this class seems to have some use cases.